ledger-service/http-json/BUILD.bazel

```
@@ -192,7 +192,7 @@ daml_compile(
         "IAccount",
         "Disclosure",
     ]],
-    target = lf_version_configuration.get("latest"),  # TODO(SC #13296) remove when interfaces in default
+    target = lf_version_configuration.get("latest"),
```

I believe these should be 
```suggestion
    target = lf_version_configuration.get("default"),
```

---

Done everywhere latest was used to ensure interfaces were available.

---

Done everywhere latest was used to ensure interfaces were available.

---

thanks!